transformer_engine/pytorch/tensor/float8_blockwise_tensor.py

```diff
@@ -403,6 +403,16 @@ def reshape(self, *shape: Tuple[int]) -> Float8BlockwiseQTensor:
         # pylint: disable=missing-function-docstring
         return _ReshapeFunc.apply(self, shape)
 
+    def untyped_storage(self) -> torch.UntypedStorage:
+        data = (
+            self._rowwise_data
+            if self._rowwise_data is not None
+            else self._columnwise_data
+        )
```

Comment:
The autoformatter prefers putting this in one line.
```suggestion
        data = self._rowwise_data if self._rowwise_data is not None else self._columnwise_data
```